messaging/build.gradle

```diff
@@ -61,7 +61,7 @@ android {
 dependencies {
 
     implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
-    implementation 'androidx.core:core-ktx:1.3.2'
+    implementation 'androidx.core:core-ktx:1.6.0'
```

Comment:
Since we kept the original version of Kotlin, do we need to/want to upgrade these?

Reply:
im gonna do a rollback to the previous version on `build.gradle`